CRM/Volunteer/BAO/Project.php

```diff
@@ -231,11 +239,14 @@ static function retrieveByID($id) {
    * @access public
    */
   public static function dataExists($params) {
```

Comment:
I think this could have a better name. Sounds like a duplicate check function. "validateProjectParams"